server/db/UnoDB.go

```diff
@@ -34,6 +34,8 @@ type UnoDB interface {
 	SavePlayer(model.Player) error
 	// Adds a Players message to the db
 	AddMessage(gameID string, playerID string, message model.Message) (*model.Game, error)
+	// updates the message in the db
+	UpdateNotification(gameID string, notification model.Notification) (*model.Game, error)
```

Comment:
Why is the notification being stored in the database at all? Shouldn't this be a purely front end thing?

Reply:
There is a front and backend aspect. When you call the snackService.notifyAll() it will notify every player in the current game game with the given notification. I tried lots of different things but this was the only way I was able to get the notification to send to everyone.